demos/sample_azure_iot_adu/sample_azure_iot_pnp_simulated_data.c

```diff
@@ -23,6 +23,8 @@
 
 /*-----------------------------------------------------------*/
 
+#define sampleazureiotUPDATE_HANDLER                      "microsoft/swupdate:1"
```

Comment:
```suggestion
#define sampleazureiotUPDATE_HANDLER                      "contoso/swupdate:1"
```

Reply:
The handler name as is an ADU defined term